src/meshlink.c

```diff
@@ -54,6 +54,8 @@ __thread meshlink_errno_t meshlink_errno;
 meshlink_log_cb_t global_log_cb;
 meshlink_log_level_t global_log_level;
 
+typedef bool (*search_node_by_condition_t)(node_t *, void *);
```

Comment:
Since this function is not supposed to change either the node or the condition, you should make both parameters `const`.

Reply:
The parameters have been changed to const